### PR TITLE
Pangle 3.7.1.3.0 Adapter

### PR DESCRIPTION
**Pull request by Clyde19-z**
update rewardVerify callback

### CODE REVIEW

File: Pangle/src/main/java/com/mopub/mobileads/PangleAdBanner.java [hunk 1]
```diff
@@ -102,7 +102,6 @@ protected void load(@NonNull final Context context, @NonNull final AdData adData
 
         final AdSlot.Builder adSlotBuilder = new AdSlot.Builder()
                 .setCodeId(mPlacementId)
-                .setSupportDeepLink(true)
```
Comment (p158276) [inline Pangle/src/main/java/com/mopub/mobileads/PangleAdBanner.java:105]: Is SupportDeepLink no longer supported from Pangle SDK? What is the difference between the previous version and the current one?

File: Pangle/src/main/java/com/mopub/mobileads/PangleAdapterConfiguration.java [hunk 2]
```diff
@@ -154,8 +154,6 @@ public static void pangleSdkInit(Context context, String appId) {
             TTAdSdk.init(context, new TTAdConfig.Builder()
                     .appId(appId)
                     .useTextureView(hasWakeLockPermission)
-                    .appName(MOPUB_NETWORK_NAME)
-                    .allowShowPageWhenScreenLock(sIsAllowAdShowInLockScreen)
```
Comment (p158276) [inline Pangle/src/main/java/com/mopub/mobileads/PangleAdapterConfiguration.java:158]: Is it removed from your latest SDK as well? What is the workaround for the publishers if they require this feature after this version?

File: Pangle/CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+
   * 3.6.0.4.1
      * Optimize banner size mapping.
 
   * 3.6.0.4.0
      * This version of the adapters has been certified with Pangle 3.6.0.4 and MoPub SDK 5.17.0.
 
   * 3.5.1.1.0
-     * This version of the adapters has been certified with Pangle 3.5.1.1 and MoPub SDK 5.17.0.
+     * This version of the adapters has been certifi`ed with Pangle 3.5.1.1 and MoPub SDK 5.17.0.
```
Comment (p158276) [inline Pangle/CHANGELOG.md:12]: Typo

File: Pangle/src/main/java/com/mopub/mobileads/PangleAdBanner.java (hunk 1 above)
Comment (Clyde19-z) [reply Pangle/src/main/java/com/mopub/mobileads/PangleAdBanner.java:105]: This interface is no longer support to config by developers, Pangle SDK will enable deeplink support by default.

File: Pangle/src/main/java/com/mopub/mobileads/PangleAdapterConfiguration.java (hunk 2 above)
Comment (Clyde19-z) [reply Pangle/src/main/java/com/mopub/mobileads/PangleAdapterConfiguration.java:158]: For appName:  This interface is no longer supported to config by developers, Pangle SDK will read the real app name configed on App's manifest by default.
For allowShowPageWhenScreenLock: This interface is deprecated due to policy and no longer support.

**Pull request merged**